.gitignore

```diff
@@ -39,3 +39,5 @@ qnvim.pro.user*
 CMakeLists.txt.user*
 
 tags
+
+build*/
```

Comment:
Shouldn't it be `build/`? Are we going to have directory names starting with "build" that should be ignored?

Reply:
Yes. It may be the case in Windows, where you want to build either with MSVC or MinGW compiler, so the directors could be `build-MSVC` and `build-MinGW`.